android-components/components/browser/state/src/main/java/mozilla/components/browser/state/selector/Selectors.kt

```diff
@@ -106,6 +109,23 @@ fun BrowserState.findNormalOrPrivateTabByUrl(url: String, private: Boolean): Tab
     return tabs.firstOrNull { tab -> tab.content.url == url && tab.content.private == private }
 }
 
+/**
+ * Finds and returns the tab with the given url ignoring the fragment/reference part of the url.
```

Comment:
It's not clear to me what a "fragment/reference" part of the URL is. Perhaps you can just expand the comment to include `(eg, xxxxx)`

Reply:
```suggestion
 * Finds and returns the tab with the given url ignoring the fragment identifier part of the url.
```

Just looked this up and perhaps we can just clean this up to say fragment identifier for extra clarity since that's it's formal name
https://url.spec.whatwg.org/#concept-url-fragment
https://www.w3.org/DesignIssues/Fragment.html